src/pages/en/install/manual.mdx

```diff
@@ -4,21 +4,23 @@ description: 'How to install Astro manually with NPM, PNPM, or Yarn.'
 layout: ~/layouts/MainLayout.astro
 i18nReady: true
 ---
+import Button from '~/components/Button.astro'
 import FileTree from '~/components/FileTree.astro';
-import InstallGuideTabGroup from '~/components/TabGroup/InstallGuideTabGroup.astro';
 import PackageManagerTabs from '~/components/tabs/PackageManagerTabs.astro'
 
 
-Ready to install Astro? Follow our automatic or manual set-up guide to get started.
+Ready to install Astro? Follow this manual set-up guide to get started.
+
+#### Want a quick way to get started?
```

Comment:
```suggestion
#### Want to let Houston do the work for you instead?
```
Too cheesy?  :sweat_smile:

Reply:
> Want to let our automated assistant Houston do the work for you instead?

As I wrote that, I was like, "ugh, I hate automated assistants" so NOT that.

But maybe something like: Want a guided CLI experience?

Reply:
If we want comparative maybe just?
```suggestion
#### Want a quicker way to get started?
```

Reply:
I don’t think the CLI part is really the important thing, it’s more the speed/convenience angle that’s attractive. Plus “CLI wizard” is in the button text. I thought “quick and easy” initially but usually avoid “easy” if I can.

Reply:
Yeah, my objection was that I'm not sure this clearly enough says that it's a REPLACEMENT for doing the steps below. For someone who doesn't know any better, this could be a way to start THE STEPS BELOW.

Reply:
> Prefer a guided installation experience?

Reply:
Ahhh, missed that nuance. How about…

```suggestion
#### Want a quicker way to get started instead?
```

Reply:
Ooh, I do like “prefer” to help communicate this.

Reply:
Maybe?

```suggestion
#### Prefer a quicker way to get started?
```